content/updates-to-ember-data-versioning-strategy.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Updates To The EmberData Lockstep Versioning Strategy
+authors:
+  - chris-thoburn
+  - jared-galanis
+  - the-ember-data-team
+date: 2023-10-31T00:00:00.000Z
+tags:
+  - '2023'
+  - ember-data
+---
+
+# Updates To The EmberData Lockstep Versioning Strategy
+
+Today the EmberData team is excited to announce changes to how we manage lockstep
+versioning with the broader Ember.js project.
+
+The primary goals of these changes to improve version-to-version stability for our consumers.
+
+First, for the project to release fewer versions that contain no effective changes other than the version bump. We believe this will help consumers navigate version-by-version and lts-by-lts upgrades with greater stability.
```

Comment:
```suggestion
First, by releasing fewer versions that contain no effective changes other than the version bump. We believe this will help consumers navigate version-by-version and lts-by-lts upgrades with greater stability.
```